### PR TITLE
Get twoFA property based on auth_mode settings

### PR DESCRIPTION
**Pull request by CPatchane**
Don't use a custom two_fa property anymore

### CODE REVIEW

**Comment by CPatchane** [comment]
[[Settings] 2FA marqué comme désactivé dans settings alors que c'est actif côté stack](https://trello.com/c/1vtVWiN5/927-settings-2fa-marqu%C3%A9-comme-d%C3%A9sactiv%C3%A9-dans-settings-alors-que-cest-actif-c%C3%B4t%C3%A9-stack)

**Comment by nono** [comment]
Ping @gregorylegarec

File: src/actions/twoFactor.js [hunk 1]
```diff
@@ -75,8 +75,11 @@ export const checkTwoFactorCode = (code, mode = AUTH_MODE.TWO_FA_MAIL) => {
       auth_mode: mode
     })
       .then(() => {
-        dispatch({ type: CHECK_TWO_FACTOR_CODE_SUCCESS })
-        dispatch(updateInfo('two_fa', true))
+        dispatch({
+          type: CHECK_TWO_FACTOR_CODE_SUCCESS,
+          field: 'auth_mode',
```
Comment (gregorylegarec) [inline src/actions/twoFactor.js]: Is `auth_mode` a new field ? If so I think we should use `authMode` instead to be compliant with recent guidelines.

Comment (CPatchane) [reply src/actions/twoFactor.js]: This is a field based directly on the stack name, it's why this is not camelCase here

**Pull request merged**